platform/barefoot/sonic-platform-modules-imt/sonic_platform/__init__.py

```diff
@@ -0,0 +1,2 @@
+__all__ = ['chassis', 'psu', 'thermal', 'sensors', 'sfp']
```

Comment:
Why exactly these modules we want to import with "import *" ?

Reply:
they are used in external demons.